docs/v1/models.md

```diff
@@ -34,18 +34,18 @@
 + T: transaction
 + H 
     + version: `` - Transaction version
-    + created: `2018-01-01T00:00:00.000000000Z`
+    + created: `2018-01-01T00:00:00.000000000Z` - transaction generated time.
     + hash: `2g3ZSrEnsUWeX5Mxz5uTh2b4KVpVQS7Ek2HzZd759FHn`
     + signature: `3oWmCMNHExRQnZVEBSH16ZBgLE6ayz7t1fsjzTjAB6WpXMpkDJbhcL8KudqFFG21XmfSXnJH1BLhnBUh4p68yFeR`
 + B
-    + source: GDIRF4UWPACXPPI4GW7CMTACTCNDIKJEHZK44RITZB4TD3YUM6CCVNGJ
-    + fee: 10000
-    + sequenceID: 1
+    + source: GDIRF4UWPACXPPI4GW7CMTACTCNDIKJEHZK44RITZB4TD3YUM6CCVNGJ - Account's Public address that send BOSCoin to other account. 
```

Comment:
We may decided to use `BOScoin` instead of `BOSCoin`

Reply:
alright.